src/Ninja/G1CP/Content/Misc/constant.d

```diff
@@ -0,0 +1,13 @@
+/*
+ * Replace element in string array constant
+ */
+func int G1CP_ConstReplaceStr(var string strConst, var string originalStr, var string correctedStr) {
```

Comment:
We might need a different function name here, as the current name is not very descriptive. The function is **searching** a string **array** constant for a particular string element and replaces it **if it exists**. I don't have name in mind but we can discuss here.

Reply:
Sounds good! I would add another `e` or `em` in the end for `G1CP_ReplaceStringConstArrEle` or `1CP_ReplaceStringConstArrElem`. Thoughts?

Reply:
I would go with `G 1CP_ReplaceStringConstArrElem` then.